src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int DEFAULT_OPERATIONS_NUMBER = 2;
+    private static final String FIRST_OPERATION_NAME = "supply";
+    private static final String SECOND_OPERATION_NAME = "buy";
+    private static final String RESULT_OPERATION_NAME = "result";
+    private static final String SYMBOL_COMMA = ",";
+    private static final String SYMBOL_LINE_SEPARATOR = "\n";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File myFile = new File(toFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(myFile))) {
+            bufferedWriter.write(calculateReport(readFile(fromFileName)));
```

Comment:
calculateReport() and readFile() don't seem to be able to produce IOException so there is no point in putting them inside the try-catch

Reply:
It is necessary to add try/catch because both new FileWriter and write method can potencially produce an exception